test/run

```diff
@@ -26,6 +26,9 @@ case $TEST_SCENARIO in
         test/image-prepare --verbose --overlay $TEST_OS
         test/common/run-tests --jobs ${TEST_JOBS:-1} --test-dir test/verify
         ;;
+    mobile)
+        echo "Deprecated test scenario: $TEST_SCENARIO"
```

Comment:
We don't need this any more, it was just dropped from bots.

Reply:
Oh, rebase mistake.